create-pull-request.py

```diff
@@ -20,6 +20,9 @@ def ignore_event(github_event):
     # not to commit the changes. They close the PR and delete the branch. This creates a 
     # "push" event that we want to ignore, otherwise it will create another branch and PR on
     # the same commit.
+    if 'schedule' in github_event:
```

Comment:
If I can request one small change. Please shift the code you added above the comments about ignoring push events on deleted branches. That comment block directly relates to the `if deleted == "True":` line.
Like this:
```
    if 'schedule' in github_event:
        print("Allow schedule event.")
        return False
    # Ignore push events on deleted branches
    # The event we want to ignore occurs when a PR is created but the repository owner decides
    # not to commit the changes. They close the PR and delete the branch. This creates a 
    # "push" event that we want to ignore, otherwise it will create another branch and PR on
    # the same commit.
    deleted = "{deleted}".format(**github_event)
    if deleted == "True":
        print("Ignoring delete branch event.")
```

Reply:
Done. I'm glad to see my first Python Pull Request getting merged 😄